testapi.pm

```diff
@@ -936,9 +936,10 @@ execution to complete.
 C<$output> can be used as an explanatory text that will be displayed with the execution of
 the command.
 
-With C<die_on_timeout> -1 (default) a warning will be printed to log. To avoid
-this warning, set it explicit to 0. With C<die_on_timeout> equal to 1, this command
-throw an exception, if timeout expires.
+By default C<script_run> will throw an exception if timeout expires. This is
```

Comment:
```suggestion
By default C<script_run> will throw an exception if the timeout is exceeded. This is
```
```suggestion
By default C<script_run> will throw an exception if the timeout has expired. This is
```
If you don't like the "the":
```suggestion
By default C<script_run> will throw an exception if C<timeout> has expired. This is
```

Reply:
ok, I went with "the timeout has expired"